src/pages/trade/ui/route-book.tsx

```diff
@@ -126,70 +11,56 @@ const RouteBookLoadingState = () => {
   );
 };
 
-const RouteBookData = observer(
-  ({ baseAsset, quoteAsset }: { baseAsset: Metadata; quoteAsset: Metadata }) => {
-    const asset1Exponent = getDisplayDenomExponent(baseAsset);
-    const asset2Exponent = getDisplayDenomExponent(quoteAsset);
-    const { data: computePositionId } = useComputePositionId();
-    const { data } = useBook(baseAsset.symbol, quoteAsset.symbol, 100, 50);
-    const asks = getDisplayData({
-      data: data?.asks ?? [],
-      computePositionId,
-      asset1: baseAsset,
-      asset2: quoteAsset,
-      isBuySide: false,
-      limit: 8,
-    });
-    const bids = getDisplayData({
-      data: data?.bids ?? [],
-      computePositionId,
-      asset1: baseAsset,
-      asset2: quoteAsset,
-      isBuySide: true,
-      limit: 8,
-    });
-
-    return (
-      <div className='h-[512px] text-white'>
-        <table className='w-full'>
-          <thead>
-            <tr>
-              <th>Price</th>
-              <th className='text-right'>Amount</th>
-              <th className='text-right'>Total</th>
+const RouteBookData = observer(({ bookData: { multiHops } }: { bookData: RouteBookResponse }) => {
+  return (
+    <div className='h-[512px] text-white'>
+      <table className='w-full'>
+        <thead>
+          <tr>
+            <th>Price</th>
+            <th className='text-right'>Amount</th>
+            <th className='text-right'>Total</th>
+            <th className='text-right'>Hops</th>
+          </tr>
+        </thead>
+        <tbody>
+          {multiHops.sell.map(trace => (
+            <tr key={trace.price + trace.total} style={{ color: 'red' }}>
+              <td className='text-left tabular-nums'>{trace.price}</td>
+              <td className='text-right tabular-nums'>{trace.amount}</td>
+              <td className='text-right tabular-nums'>{trace.total}</td>
+              <td className='text-right tabular-nums'>{trace.hops.length}</td>
+            </tr>
+          ))}
+          {multiHops.buy.map(trace => (
+            <tr key={trace.price + trace.total} style={{ color: 'green' }}>
+              <td className='text-left tabular-nums'>{trace.price}</td>
+              <td className='text-right tabular-nums'>{trace.amount}</td>
+              <td className='text-right tabular-nums'>{trace.total}</td>
+              <td className='text-right tabular-nums'>{trace.hops.length}</td>
```

Comment:
Hops now visible. Final designs can be implemented in follow up PR.

<img width="387" alt="Screenshot 2024-10-30 at 2 55 36 PM" src="https://github.com/user-attachments/assets/8cf31029-4b97-4464-85db-7c53086c88de">